modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi_add.lua

```diff
@@ -16,7 +16,7 @@ if not iw then
 	return
 end
 
-m = SimpleForm("network", translate("Join Network: Settings"))
+m = SimpleForm("network", translate("Joining Network: ") .. http.formvalue("join"))
```

Comment:
Can you use a placeholder here? This makes it easier to translate into other languages, e.g.
`translate("Joining Network %q", http.formvalue("join"))`